game_eggs/steamcmd_servers/arma/arma3/egg-arma3.json

```diff
@@ -1,60 +1,60 @@
 {
-    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "_comment": "Pterodactyl Arma 3 Egg ~ David Wolfe (Red-Thirten) ~ 2021-07-11",
     "meta": {
         "version": "PTDL_v1",
-        "update_url": null
+        "update_url": "https:\/\/github.com\/parkervcp\/eggs\/blob\/master\/game_eggs\/steamcmd_servers\/arma\/arma3\/egg-arma3.json"
     },
-    "exported_at": "2021-07-02T04:17:45+03:00",
     "name": "Arma 3",
     "author": "rehlmgaming@gmail.com",
     "description": "Experience true combat gameplay in a massive military sandbox. Deploying a wide variety of single- and multiplayer content, over 20 vehicles and 40 weapons, and limitless opportunities for content creation, this is the PC's premier military game. Authentic, diverse, open - Arma 3 sends you to war.",
     "features": null,
     "images": [
+        "ghcr.io\/parkervcp\/yolks:arma3",
```

Comment:
Your image is hosted at `ghcr.io/parkervcp/games:arma3` (Parker has to make the game image packages public first)

Reply:
That's what I thought it would be too, but @parkervcp [stated in Discord](https://discord.com/channels/122900397965705216/404105683336953866/864613123490316318) it would be `ghcr.io\/parkervcp\/yolks:<tag>`, and I didn't want to go against that. Regardless, fixed: bd311a6